ros/src/sensing/filters/packages/points_preprocessor/nodes/points_concat_filter/points_concat_filter.cpp

```diff
@@ -0,0 +1,96 @@
+/*
+ * fusion_filter.cpp
+ *
+ * Created on   : September 4, 2017
+ * Author       : Akihito OHSATO
+ */
+
+#include <ros/ros.h>
+#include <sensor_msgs/PointCloud2.h>
+#include <message_filters/subscriber.h>
+#include <message_filters/synchronizer.h>
+#include <message_filters/sync_policies/approximate_time.h>
+#include <pcl_ros/point_cloud.h>
+#include <pcl_ros/transforms.h>
+#include <pcl_conversions/pcl_conversions.h>
+#include <pcl/point_types.h>
+#include <velodyne_pointcloud/point_types.h>
+#include <tf/tf.h>
+#include <tf/transform_listener.h>
+
+class PointsConcatFilter
+{
+public:
+  PointsConcatFilter();
+
+private:
+  typedef pcl::PointXYZI PointT;
+  typedef pcl::PointCloud<PointT> PointCloudT;
+  typedef sensor_msgs::PointCloud2 PointCloudMsgT;
+  typedef message_filters::sync_policies::ApproximateTime<PointCloudMsgT, PointCloudMsgT> SyncPolicy;
+  ros::NodeHandle nh_, pnh_;
```

Comment:
I recommend to use more significant names. (i.e. `node_handle_`, `private_handle_`).